setup.cfg

```diff
@@ -1,14 +1,72 @@
+[metadata]
+name = DecayLanguage
+description = A language to describe particle decays, and tools to work with them.
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/scikit-hep/decaylanguage
+author = Henry Fredrick Schreiner III, Eduardo Rodrigues
+author_email = henry.schreiner@cern.ch, eduardo.rodrigues@cern.ch
+maintainer = The Scikit-HEP admins
+maintainer_email = scikit-hep-admins@googlegroups.com
+license = BSD-3-Clause
+license_file = LICENSE
+platforms = Any
+classifiers =
+    Development Status :: 4 - Beta
+    Intended Audience :: Developers
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 2
+    Programming Language :: Python :: 2.7
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Topic :: Scientific/Engineering
+keywords =
+    HEP
+    particle
+    decay
+    chain
+    representation
+
+[options]
+packages = find:
+install_requires =
+    attrs>=19.2
+    graphviz
+    lark-parser>=0.8.0,<0.8.6
```

Comment:
We need to fix this to work with newer versions...

Reply:
Indeed. I look at that a while back and concluded at the time that Lark had changed behaviour significantly, in a way that almost looked like a bug. I can't remember the details now but then decided not to dwell on it immediate. I agree that we need to move to recent versions, also to potentially make things work for latest Python versions. I will create an issue ...